1.0/docs/devguide/properties.md

```diff
@@ -524,44 +496,68 @@ Example:
 
     });
 
-**Array mutation methods.** Observing changes to arrays is dependent on the change to the array
-being made through one of the [array mutation methods](#array-mutation) provided
-on Polymer elements, which provides the required notification to elements with
-registered interest.
-{: .alert .alert-info }
+### Deep sub-property observation {#deep-observation}
 
-When you specify a wildcard path on an array, the observer is for both splices as
-well as array element sub-property changes.  So the  observer in the
-following example will be called for all additions, removals, and deep changes
-that occur in the array:
+To call an observer when any (deep) sub-property of an
+object or array changes, specify a path with a wildcard (`*`).
 
-    Polymer({
+When you specify a path with a wildcard, the argument passed to your
+observer is a change record object with the following properties:
 
-      is: 'x-custom',
+*   `path`. Path to the property that changed. Use this to determine whether
+    a property changed, a sub-property changed, or an array was mutated.
+*   `value`. New value of the path that changed.
+*   `base`. The object matching the non-wildcard portion of the path. 
```

Comment:
Should also mention: 

For array mutations, `path` is the path to the array that changed, followed by `.splices`. And the change record includes the `indexSplices` and `keySplices` properties described in [Observe array mutations](link)